contribution_plan/models.py

```diff
@@ -71,10 +94,10 @@ def filter(self, *args, **kwargs):
         return super(ContributionPlanBundleDetailsManager, self).filter(*args, **kwargs)
 
 
-class ContributionPlanBundleDetails(models.Model):
+class ContributionPlanBundleDetails(core_models.UUIDVersionedModel):
```

Comment:
This models create copy on save as in legacy, we don't want that, again we might create a UUIDHistoryModel in the core to implement simple history module